espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/GsonParserHelpers.kt

```diff
@@ -37,4 +38,26 @@ class GsonParserHelpers {
         }
         return defaultValue
     }
+
+    fun parsePrimitive(jsonPrimitive: JsonPrimitive): Any = when {
```

Comment:
this looks so Scal'ish %)

Reply:
Yeah that was Kazu's idea. I like it. What do you think?

Reply:
https://storage.googleapis.com/images.frenchly.us/2016/11/not-bad-obama.jpg

Reply:
Oh, so Scal'ish? (I haven't worked with Scala x) )
Actually, this is kind of functional style, I think :)